spec.md

```diff
@@ -151,6 +151,18 @@ Service Broker MAY reject the request with `412 Precondition Failed` and
 provide a message that informs the operator of the API version that is to be
 used instead.
 
+## Extending Resources
```

Comment:
I am fine with the updated text body but I think the section header reads as if the additional fields have some OSB support. How about a second title like:

## Unknown Fields

or 

## Ignored Fields

or 

## Out of Scope JSON Payloads

or 

## Ignored JSON Fields

or 

## Out of Spec Fields

or 

## Non-OSB Communications



my vote is on `Unknown Fields`